system-tests/hip/hipInfo.py

```diff
@@ -0,0 +1,26 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class HipInfo(rfm.RegressionTest):
+    valid_systems = ['lumi:gpu']
+    valid_prog_environs = ['builtin-hip']
+    modules = ['rocm']
+    build_system = 'SingleSource'
+    sourcepath = 'hipInfo.cpp'
+    executable = 'hipInfo'
+    build_locally = False
+    maintainers = ['mszpindler']
+    num_gpus_per_node = 8
+
+    @run_before('compile')
+    def set_compiler_flags(self):
+        self.build_system.cc = 'hipcc'
+        self.build_system.cxx = 'hipcc'
```

Comment:
The compiler definition maybe we can move it to the `builtin-hip` definition in the configuration?